NineChronicles.Headless.Executable/Store/NoOpStateStore.cs

```diff
@@ -0,0 +1,32 @@
+using System;
```

Comment:
```suggestion
#nullable enable
using System;
```

Reply:
I opened #349 issue to enable nullable references feature over all projects in this repository. I will enable those in next pull request. How do you think?

Reply:
👌